engine/server/engine/enclave_manager/enclave_manager.go

```diff
@@ -473,6 +473,13 @@ func (manager *EnclaveManager) getEnclavesWithoutMutex(
 
 }
 
+func (manager *EnclaveManager) Stop() error {
```

Comment:
nit: I've seen `Close` more often that `Stop`. I think in Go, the `Close()` method makes it implement the `Closer` interface, which seem to indicate it's more common to use Close.
If you feel `Stop` is better, let's go for `Stop`, no super strong opinion TBH

Reply:
I'll use `Close` at the beginning I used `Close` but then I had some doubts, but now that you are also mentioning it, I'll rollback it